libsolidity/ast/AST.cpp

```diff
@@ -165,9 +166,14 @@ vector<EventDefinition const*> const& ContractDefinition::interfaceEvents() cons
 vector<ErrorDefinition const*> ContractDefinition::interfaceErrors() const
 {
 	set<ErrorDefinition const*, CompareByID> result;
-	// TODO add all referenced errors
 	for (ContractDefinition const* contract: annotation().linearizedBaseContracts)
 		result += filteredNodes<ErrorDefinition>(contract->m_subNodes);
+	solAssert(annotation().creationCallGraph.set() == annotation().deployedCallGraph.set(), "");
```

Comment:
Now we are collecting all errors that are referenced. The problem is that this function is used too early in the "hash collision checker", at a point where we do not even have the call graph yet.